### PR TITLE
Define selectable basemaps (tile layers) #167

### PR DESCRIPTION
**Pull request by ut**

### CODE REVIEW

**Comment by ut** [comment]
Preview:

image

File: app/views/layers/_form.html.haml [hunk 1]
```diff
@@ -90,30 +90,38 @@
...
-            .large-6.medium-6.small-12.cell
+            .large-12.medium-12.small-12.cell
+              %label Predefined Basemaps
+              %select#basemaps
+                %option{value: '', "data-url": '', "data-backgroundcolor": '', "data-attribution": '' }
+                - basemaps.each_value do |b|
+                  %option{value: b['id'], "data-url": b['url'], "data-backgroundcolor": b['backgroundcolor'], "data-attribution": b['attribution'] }
+                    = b['title']
+              %p.shy
+                Please select one the prefinied basemaps
```
Comment (Sefux) [inline app/views/layers/_form.html.haml:101]: typo:   Please select one **of** the prefinied basemaps

**Review by Sefux — COMMENTED** [review]
typo

File: app/views/layers/_form.html.haml (hunk 1 above)
Comment (Sefux) [reply app/views/layers/_form.html.haml:101]: Please select one of the pre**de**finied basemaps

**Review by Sefux — APPROVED** [review]

File: app/views/layers/_form.html.haml (hunk 1 above)
Comment (ut) [reply app/views/layers/_form.html.haml]: done! thanx

**Pull request merged**